indexer/ai/src/main/java/com/google/cloudsearch/ai/AISkillSet.java

```diff
@@ -0,0 +1,66 @@
+package com.google.cloudsearch.ai;
+
+import com.sun.org.apache.bcel.internal.generic.ANEWARRAY;
+import org.checkerframework.checker.units.qual.C;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class AISkillSet {
+
```

Comment:
Add a public constructor to the class that takes the configuration file path string and internally parses the skills. We don't need to expose skills list or parse method, they should be private to the class

Reply:
Actually, parse() is a part of the interface and if it has to be a part of the interface then I cannot make it private or protected. So, the only option would be removing it from the interface, but I think we should keep it in the interface as it is common to all skills. What do you think?